test/Solver/range1.ll

```diff
@@ -0,0 +1,14 @@
+; REQUIRES: solver

```

Comment:
in this and other test cases you need a "check" line or else you're not actually checking for anything in particular to happen